docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/memberlist.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Migrating from Consul to Memberlist KV store for hash rings"
+menuTitle: "Migrating from Consul to Memberlist"
+description: "Learn how to migrate from using Consul as KV store for hash rings to using memberlist without any downtime."
+weight: 30
+---
+
+# Migrating from Consul to Memberlist as KV store for hash rings without downtime
+
+Mimir Jsonnet uses `memberlist` as KV store for hash rings, unless disabled by using
+
+```jsonnet
+{
+  _config+:: {
+    memberlist_ring_enabled: false
+  }
+}
+```
+
+If you are still using Consul (ie. you have `memberlist_ring_enabled: false` in your Jsonnet, this was default before Grafana Mimir 2.2),
+and would like to migrate to `memberlist` without any downtime, you can follow instructions in this document.
+
+## Step 1: Enable memberlist and multi KV store.
```

Comment:
Docs squad has strong opinions on how step-by-step documentation should be formatted. They want to use numbered list items. I personally disagree, because their formatting makes it very complicated to read. The format you used is way better in my opinion, but let's see what docs squad says.